as/itests/org.jboss.tools.as.jmx.ui.bot.itests/META-INF/MANIFEST.MF

```diff
@@ -1,8 +1,8 @@
 Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %BundleName
-Bundle-SymbolicName: org.jboss.tools.jmx.ui.bot.test
-Bundle-Version: 1.10.100.qualifier
+Bundle-SymbolicName: org.jboss.tools.as.jmx.ui.bot.itests
+Bundle-Version: 1.0.0.qualifier
```

Comment:
I strongly suggest you use the 3.6.2 version like in the rest of the 'as' modules.  The reason for this is because the up-version maven / tycho script will (for some reason) miss this upversion request for modules that don't all have the same bundle version. 

Test it, if you want.  Check out this topic branch, and run mvn org.eclipse.tycho:tycho-versions-plugin:set-version -DnewVersion=3.6.3-SNAPSHOT.  You will see all bundles are up-versioned EXCEPT this one. 

This must be a bug with the tycho plugin, but I've never developed or tested a tycho plugin before so I'm not at all sure how to fix it ;) It might be fun to try that out, though.

Reply:
Damn, you are right. I did it first time when I placed both modules (itests, test-framework) under jmx component. I forgot why I did it first time, today. My bad. Making things right now.